library/src/main/java/com/wultra/android/mtokensdk/operation/rejection/RejectionReason.kt

```diff
@@ -14,7 +14,7 @@
  * and limitations under the License.
  */
 
-package com.wultra.android.mtokensdk.operation
+package com.wultra.android.mtokensdk.operation.rejection
```

Comment:
This is an incompatible change. This would cause our integrators to get a compilation error when just upgrading the SDK. The package needs to stay the same

Reply:
🤦‍♂️ I see...